{"name": "test_order", "status": "broken", "statusDetails": {"message": "json.decoder.JSONDecodeError: Expecting value: line 1 column 2 (char 1)", "trace": "self = <testcase.test_mtx_order.TestOrder object at 0x000001EE662FB1D0>\n\n    def test_order(self):\n>       resp_order = self.order_object.order(self.session).json()\n\ntestcase\\test_mtx_order.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \napi\\apiOrder.py:29: in order\n    setting.JUMP_URL = resp_order.json().get('data').get('jump_url')\nd:\\python\\untitled5\\lib\\site-packages\\requests\\models.py:900: in json\n    return complexjson.loads(self.text, **kwargs)\nD:\\python\\lib\\json\\__init__.py:354: in loads\n    return _default_decoder.decode(s)\nD:\\python\\lib\\json\\decoder.py:339: in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <json.decoder.JSONDecoder object at 0x000001EE64441240>\ns = \"[Fiddler] The connection to '10.74.131.110' failed. <br />Error: TimedOut (0x274c). <br />System.Net.Sockets.SocketEx...                                                                                                                      \"\nidx = 0\n\n    def raw_decode(self, s, idx=0):\n        \"\"\"Decode a JSON document from ``s`` (a ``str`` beginning with\n        a JSON document) and return a 2-tuple of the Python\n        representation and the index in ``s`` where the document ended.\n    \n        This can be used to decode a JSON document from a string that may\n        have extraneous data at the end.\n    \n        \"\"\"\n        try:\n            obj, end = self.scan_once(s, idx)\n        except StopIteration as err:\n>           raise JSONDecodeError(\"Expecting value\", s, err.value) from None\nE           json.decoder.JSONDecodeError: Expecting value: line 1 column 2 (char 1)\n\nD:\\python\\lib\\json\\decoder.py:357: JSONDecodeError"}, "attachments": [{"name": "log", "source": "69acb9f1-cade-44d9-bd9a-2e5b82694388-attachment.txt", "type": "text/plain"}], "start": 1606733837704, "stop": 1606733858711, "uuid": "70366b94-23b2-4b55-8bb8-a5121a908373", "historyId": "22c7d134318c12d29726b1ce1f22f7d9", "testCaseId": "fd120f177de86113a56d53b5031df56f", "fullName": "testcase.test_mtx_order.TestOrder#test_order", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_mtx_order"}, {"name": "subSuite", "value": "TestOrder"}, {"name": "host", "value": "liusx17111"}, {"name": "thread", "value": "35148-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_mtx_order"}]}